recipes/CVSS/S2ST/utils/logger.py

```diff
@@ -0,0 +1,52 @@
+import wandb
```

Comment:
Ok so this should definitely go to our official logger dir. Could you please let us know what are the differences between this and the logger that we have in SpeechBrain already for WandB ?

Reply:
It will be replaced by Tensorbard